### DISCUSSION TITLE
Where is js file for Chirpy theme?

### DISCUSSION THREAD

**Discussion by seol-hwa** [cotes2020/jekyll-theme-chirpy#1593] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

There is no js/dist folder in my downloads.
This causes an error in the terminal and prevents Git from building.

### Operations you have already tried

1. I took the entire file from someone else who was using this theme and used it.
2. Use a different theme.

**Comment by kungfux** [comment]
Please refer to the [development guide](https://github.com/cotes2020/jekyll-theme-chirpy/wiki/Development-&-Test-Environments#external-javascript) and make sure you have executed `npm run build` or `npm run watch` command.

> run the npm run build command. The new compressed JS files will be exported to assets/js/dist/
